frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -610,20 +610,33 @@ const TAGGED_TEMPLATES = {
   disabledWindowsMdm: (activity: IActivity) => {
     return <> told Fleet to turn off Windows MDM features.</>;
   },
+  // TODO: Combine ranScript template with host details page templates
+  // frontend/pages/hosts/details/cards/Activity/PastActivity/PastActivity.tsx and
+  // frontend/pages/hosts/details/cards/Activity/UpcomingActivity/UpcomingActivity.tsx
   ranScript: (
     activity: IActivity,
     onDetailsClick?: (type: ActivityType, details: IActivityDetails) => void
   ) => {
+    const { script_name, host_display_name, script_execution_id } =
+      activity.details || {};
     return (
       <>
         {" "}
-        ran a script on {activity.details?.host_display_name}.{" "}
+        told Fleet to run{" "}
```

Comment:
```suggestion
        ran{" "}
```

@gillespi314 Sorry for last minute change, but talked with Noah now and we would like to change this. I made changes in [Figma](https://www.figma.com/file/Ab3yKMQJMjmVSUWJMwlubP/%2315529-Upcoming-activities%3A-Target-scripts-on-hosts-whether-they're-online-or-offline?type=design&node-id=2-130&mode=design&t=xhRFpmhAFfzljWNr-11) as well

Reply:
@marko-lisica, no problem! Just to be sure we're on the same page:

There is similar text for script activities in three places in the UI:
- Dashboard activity feed
- Host past activities
- Host upcoming activities

Are we using the same template in all three places? If not, could you give me examples for each of those? I'm just trying to be sure that I understand where Figma is intentionally diverging vs. where there may be unintended artifacts of prior designs.

Reply:
Here are examples: https://www.figma.com/file/Ab3yKMQJMjmVSUWJMwlubP/%2315529-Upcoming-activities%3A-Target-scripts-on-hosts-whether-they're-online-or-offline?type=design&node-id=663-5788&mode=design&t=haJheQBeCwiwLcsa-11

Reply:
@gillespi314 a heads-up, would you want to merge this and address this afterwards or do it as part of this PR?

Reply:
sorry, I submitted this and you already took care of it